x-pack/packages/kbn-ecs-data-quality-dashboard/impl/data_quality/data_quality_panel/body/data_quality_details/storage_details/helpers.test.ts

```diff
@@ -0,0 +1,382 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import numeral from '@elastic/numeral';
+import { euiThemeVars } from '@kbn/ui-theme';
+
+import { EMPTY_STAT } from '../../../../helpers';
+import {
+  DEFAULT_INDEX_COLOR,
+  getFillColor,
+  getFlattenedBuckets,
+  getGroupFromPath,
+  getLayersMultiDimensional,
+  getLegendItems,
+  getLegendItemsForPattern,
+  getPathToFlattenedBucketMap,
+  getPatternLegendItem,
+  getPatternSizeInBytes,
+} from './helpers';
+import { alertIndexWithAllResults } from '../../../../mock/pattern_rollup/mock_alerts_pattern_rollup';
+import { auditbeatWithAllResults } from '../../../../mock/pattern_rollup/mock_auditbeat_pattern_rollup';
+import { packetbeatNoResults } from '../../../../mock/pattern_rollup/mock_packetbeat_pattern_rollup';
+import { PatternRollup } from '../../../../types';
+
+const defaultBytesFormat = '0,0.[0]b';
+const formatBytes = (value: number | undefined) =>
+  value != null ? numeral(value).format(defaultBytesFormat) : EMPTY_STAT;
+
+const ilmPhases = ['hot', 'warm', 'unmanaged'];
+const patterns = ['.alerts-security.alerts-default', 'auditbeat-*', 'packetbeat-*'];
```

Comment:
Nit and def not necessary, but with discussions around alerts index name changes, may be better to use the `${DEFAULT_ALERTS_INDEX}-default` here.

Reply:
i'm reluctant to use `${DEFAULT_ALERTS_INDEX}-default` in the context of these tests (and others) because artifacts of the current value, `.alerts-security.alerts-default`, are ubiquitous in the mock (`stats` and `ILM`) data, and in most of the tests.

In most contexts, there's nothing magic about `.alerts-security.alerts-default` other than it's currently a "foreign key" in the mock data. If the value behind the `${DEFAULT_ALERTS_INDEX}-default` changes, tests may start failing (with false positives) unless the mock data is manually updated via search & replace.

Reply:
Yea, took me a sec to realize all of these are just in test files, so doesn't really matter what it is as long as it aligns with the mocks. Ignore me...I've been seeing the index pattern everywhere so I'm very sensitive to it 😂